coxeter/shape_classes/sphere.py

```diff
@@ -93,3 +93,27 @@ def is_inside(self, points):
         """
         points = np.atleast_2d(points) - self.center
         return np.linalg.norm(points, axis=-1) <= self.radius
+
+    def compute_form_factor_amplitude(self, q, density=1.0):  # noqa: D102
+        # Use the parent docstring.
+
+        # The formula for a the form factor of a sphere may be found here:
+        # http://gisaxs.com/index.php/Form_Factor:Sphere
+        # (among other sources).[jj:w
+        q = np.atleast_2d(q)
+        form_factor = np.empty(q.shape[0], dtype=np.complex128)
+        qsq = np.sum(q * q, axis=-1)
```

Comment:
Would prefer a consistent name with previous computations of the same thing.
```suggestion
        q_sqs = np.sum(q * q, axis=-1)
```